src/main/java/jdbchomework/entity/AbstractEntity.java

```diff
@@ -39,10 +39,16 @@ public String getName() {
     }
 
     public void setName(String name) {
+        if (name == null || name.equals("")) {
```

Comment:
All checks should be made in a console layer. Setter should receive verified input and just set it.